tailscale/client_test.go

```diff
@@ -125,6 +125,7 @@ func TestACL_Unmarshal(t *testing.T) {
 			Name:       "It should handle HuJSON ACLs",
 			ACLContent: huJSONACL,
 			UnmarshalFunc: func(b []byte, v interface{}) error {
+				b = append([]byte{}, b...)
```

Comment:
Stolen from https://github.com/tailscale/tailscale-client-go/pull/86/commits/1f431cc81c00b818edffa87195f0abeeb086e3ea as the tests kept failing, might be worth throwing up in a separate quick PR @oxtoacart